.github/workflows/cli-binary-pr.yaml

```diff
@@ -0,0 +1,45 @@
+name: Build wgc binaries
+on:
+  pull_request:
+    paths:
+      - "cli/**/*"
+      - "connect/**/*"
+      - ".github/workflows/cli-ci.yaml"
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  wgc-releases-matrix:
```

Comment:
if you call it `build_test` it would be not mergable in case of an error

Reply:
done